common/src/main/java/org/jboss/hal/testsuite/page/config/ResourceAdaptersPage.java

```diff
@@ -26,24 +31,45 @@ public ResourceAdaptersConfigArea getConfigArea() {
     }
 
     public ResourceAdaptersPage navigate2ra(String raName) {
-        getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL, raName).selectRow()
-            .invoke(FinderNames.VIEW);
+        Row row = getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL, raName).selectRow();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        row.invoke(FinderNames.VIEW);
         Application.waitUntilVisible();
         return this;
     }
 
     public ResourceAdapterWizard addResourceAdapter() {
-        getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL).selectColumn().invoke(FinderNames.ADD);
+        Column column = getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL).selectColumn();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        column.invoke(FinderNames.ADD);
         return Console.withBrowser(browser).openedWizard(ResourceAdapterWizard.class);
     }
 
     public WindowState removeRa (String raName) {
-        getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL, raName).selectRow()
-            .invoke(FinderNames.REMOVE);
+        Row row = getSubsystemNavigation(RA_SUBSYSTEM_LABEL).addAddress(RA_COLUMN_LABEL, raName).selectRow();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        row.invoke(FinderNames.REMOVE);
         return Console.withBrowser(browser).openedWindow(ConfirmationWindow.class).confirm();
     }
 
     public ConfigFragment switchConfigAreaTabTo(String tabLabel) {
         return getConfig(ConfigAreaFragment.class).switchTo(tabLabel);
     }
+
+    public void navigate() {
+        FinderNavigation navigation;
+        if (ConfigUtils.isDomain()) {
+            navigation = new FinderNavigation(browser, DomainConfigEntryPoint.class)
+                    .addAddress(FinderNames.CONFIGURATION, FinderNames.PROFILES)
+                    .addAddress(FinderNames.PROFILE, ConfigUtils.getDefaultProfile())
+                    .addAddress(FinderNames.SUBSYSTEM, "Resource Adapters");
+
+        } else {
+            navigation = new FinderNavigation(browser, StandaloneConfigEntryPoint.class)
+                    .addAddress(FinderNames.CONFIGURATION, FinderNames.SUBSYSTEMS)
+                    .addAddress(FinderNames.SUBSYSTEM, "Resource Adapters");
+        }
+        navigation.selectColumn();
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+    }
```

Comment:
`navigate` method content can be replaced with 

> getSubsystemNavigation("Resource Adapters").selectColumn();
> Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();

But the main objection is that this new method is never used so useless IMO.